src/nwb_datajoint/common/common_ephys.py

```diff
@@ -74,6 +74,11 @@ def make(self, key):
                         break
             if 'probe_type' not in key:
                 key['probe_type'] = 'unknown-probe-type'
+            # Define target_hemisphere based on targeted x coordinate
+            if electrode_group.targeted_x >= 0:  # if positive or zero x coordinate
+                key["target_hemisphere"] = "right"  # define target location as right hemisphere
+            elif electrode_group.targeted_y < 0:  # if negative x coordinate
```

Comment:
Should this be changed to `electrode_group.targeted_x`? Does x correspond to the mediolateral axis?

Reply:
Yes; thanks Kyu.